api.bs

```diff
@@ -271,20 +271,20 @@ which might be used to differentiate between different outcomes.
 The value that is allocated to impressions
 is called a <dfn>conversion value</dfn>.
 A higher conversion value might be used for larger purchases
-or any outcome that is more highly-valued.
+or any outcome that is more highly valued.
 A conversion value might also be split between multiple impressions
 to split credit,
 though this capability is not presently supported in the API.
 
+<!-- TODO: This list seems like a non sequitur -->
```

Comment:
I _think_ these are meant to be TODOs themselves. I would prefer just deleting and filing an issue to make sure the spec discusses these aspects, or commenting them out with an inline todo